### ISSUE TITLE
The branding option does not work. & GDevelop logo to.

### ISSUE THREAD

**Issue by AlexMasterbober** [4ian/GDevelop#5859]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/4ian/GDevelop/issues)

### Describe the bug

after the upgrade to 5.2.179 - all types of logo and branding have stopped working.
I see in Desktop & in Web platforms - everyware the same bug. All kind of logo does not working. It just doesn't appear.
![gdevelop_logo bug](https://github.com/4ian/GDevelop/assets/147695106/8038ad48-566d-4bff-a157-7bea6d96605e)

and i tested in web platform in one of the exaples - and it doesn't work to. So problem not in my platform. 

in version 5.2.176 it works.

### Steps to reproduce

1. Go to Game settings
2. Properties
3. Load the background image or turn on Gdevelop Logo
4. It just doesn't appear in preview and final HTML5 (empty)

### GDevelop platform

Desktop

### GDevelop version

5.3.179 (editor full version: 5.3.179-e0df9b0ff39dcea7a28e6baeb1de814b8acd3399, core version: 4.0.99-0-release)

### Platform info

Windows 10 Pro, 21H2

**Comment by AlexMasterbober** [comment]
I solved the problem by replacing the file "pixi-image-manager.js" (in HTML5 game) from the previous compilation with version 5.2.176. Everything seems to be ok. Maybe this information will help you solve the problem faster.

**Comment by AlexandreSi** [comment]
Thanks for the report and sorry for the inconvenience, this is quite unexpected.
I'll have a look!

**Comment by AlexandreSi** [comment]
I can reproduce the fact that a custom logo is not displayed but the GDevelop logos are working fine for me.
If GDevelop logos do not display, could you open the develop tools (Ctrl + Shift + i) in the preview and see if there are any messages there?

**Comment by AlexMasterbober** [comment]
Logo GDevelop not working on my platform.
![image](https://github.com/4ian/GDevelop/assets/147695106/0d6bfd04-1704-4794-935c-0632d62291aa)

**Comment by AlexMasterbober** [comment]
![image](https://github.com/4ian/GDevelop/assets/147695106/975160f7-fbc5-4fca-8381-0a6f8e41b91f)

**Comment by AlexMasterbober** [comment]
in last version all works. But in Console we have this:
This is screen for my last version compilation (in 5.2.176)
Maybe it halps. Look in file "pixi-image-manager.js" it differs from the previous compiled version

![image](https://github.com/4ian/GDevelop/assets/147695106/e7ac99c3-6a16-4310-afc2-8a91f9017ecc)

**Comment by AlexandreSi** [comment]
> Logo GDevelop not working on my platform.

Once the console is opened, please hit refresh (Ctrl + R) to reload the game and get all the messages starting from the game initialization. We should see more data

**Comment by AlexMasterbober** [comment]
![image](https://github.com/4ian/GDevelop/assets/147695106/8341590c-fe99-418b-8517-e9f05fc10b13)

**Comment by AlexandreSi** [comment]
Could you send us your project to see if the same issue happens on our devices?

**Comment by AlexMasterbober** [comment]
Everything works with this file:
[pixi-image-manager(5.2.176).zip](https://github.com/4ian/GDevelop/files/13240126/pixi-image-manager.5.2.176.zip)

But not with this one:
[pixi-image-manager(5.2.179).zip](https://github.com/4ian/GDevelop/files/13240399/pixi-image-manager.5.2.179.zip)

**Comment by AlexMasterbober** [comment]
No need. I created a new compiled WEB and looked in the browser - the QDEVELOP logo works there. I also tested my project in with the KDEVELOP logo and without my logo in the browser too. It's just that the Qdevelop logo is not visible in the preview. So only my logo doesn't work.

**Comment by AlexandreSi** [comment]
Ok so I'm closing this issue, I'll reopen it if we see others encounter the same issue.

**Comment by AlexMasterbober** [comment]
No. So the branded logo does not work. Don't you want to fix it?

**Comment by AlexMasterbober** [comment]
This option not working at all. Ned to fix it.
![image](https://github.com/4ian/GDevelop/assets/147695106/da22083f-edcc-4c6f-925e-32320b055c7e)

**Comment by AlexandreSi** [comment]
Regarding your custom background, the issue is fixed.
Regarding GDevelop logos, I can't reproduce the issue so I can't continue on this topic. We'll see if other compain.

**Comment by AlexMasterbober** [comment]
Great. Will this issue be fixed in the next update? or just download the same version again?

**Comment by AlexandreSi** [comment]
In the next update that should be released soon!
